appveyor.yml

```diff
@@ -0,0 +1,54 @@
+# https://ci.appveyor.com/project/ContinuumAnalyticsFOSS/constructor
+
+branches:
+  only:
+    - master
+
+environment:
+  global:
+    PYTHON: "C:\\conda"
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+    PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
+                      # of 32 bit and 64 bit builds are needed, move this
+                      # to the matrix section.
+
+  matrix:
+    - PYTHON_VERSION: "3.6"
+      CONDA_CANARY: "false"
+    - PYTHON_VERSION: "2.7"
+      CONDA_CANARY: "false"
+    - PYTHON_VERSION: "2.7"
```

Comment:
Maybe use `3.6` for canary? This would be consistent with the Linux configuration and ... Python 2 😝  XD

Reply:
@mbargull actually, I think we should test different python versions... so we get 2.7 on windows with appveyor and 3.6 with linux.

Reply:
Ok, I briefly considered that but than my "Nah, who needs Python 2" tendency kicked in.
BTW: Glad to see this coming along!

Reply:
:-)